lib/template/datepicker.template.js

```diff
@@ -30,7 +30,7 @@ export default () => `
         PM
       </label>
     </div>
-
+    <div class="c-datepicker-mask"></div>
```

Comment:
I fix the classname.

The hand use `mouseenter` and `mouseleave` detecting your mouse in or out the `.c-datepicker__clock__hours` area - and you said mouse into the center should be revert to selecting.

So we need to mask center area for mouse into the center area.

![mask-center](https://cloud.githubusercontent.com/assets/5094008/22410193/d73ed66a-e6cf-11e6-8f26-07acfe2e5aa7.png)

You will feel no different because it is a transparent div for mask the center area